substrate/frame/staking/src/migrations.rs

```diff
@@ -64,13 +64,23 @@ type StorageVersion<T: Config> = StorageValue<Pallet<T>, ObsoleteReleases, Value
 pub mod v15 {
 	use super::*;
 
+	// The disabling strategy used by staking pallet
+	type DefaultDisablingStrategy = UpToThresholdDisablingStrategy;
```

Comment:
Should `UpToThresholdDisablingStrategy` be renamed to `UpToLimitDisablingStrategy`?

Reply:
Cosmetics but `UpToLimitDisablingStrategy` makes a bit more sense.

Reply:
Done